### PR TITLE
devfsadm: add Russian translation

### PR DESCRIPTION
**Pull request by dmorozov001**
added Russian translation

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by EmilySeville7cfg — APPROVED** [review]
- No issues found, approved.
- This page should be reviewed by `devfsadm` users.
- Thanks for a contribution! 😄

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.ru/sunos/devfsadm.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# devfsadm
+
+> Команда администрирования для `/dev`. Поддерживает пространство имен `/dev`.
+> Больше информации: <https://www.unix.com/man-page/sunos/1m/devfsadm>.
+
+- Сканировать для новых дисков:
+
+`devfsadm -c disk`
+
+- Очистить все оборванные ссылки /dev и выполнить поиск нового устройства:
```
Comment (kbdharun) [inline pages.ru/sunos/devfsadm.md:10]:
```suggestion
- Очистить все оборванные ссылки `/dev` и выполнить поиск нового устройства:
```

File: pages.ru/sunos/devfsadm.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`devfsadm -C -v`
+
+- Пробный запуск - вывод того, что бы изменилось, но без произведения модификаций:
```
Comment (kbdharun) [inline pages.ru/sunos/devfsadm.md:14]:
```suggestion
- Пробный-запуск - вывод того, что бы изменилось, но без произведения модификаций:
```

**Review by kbdharun — COMMENTED** [review]
Minor suggestions, after this the PR is GTG.

**Comment by pixelcmtd** [comment]
@dmorozov001 could you please take a look at @kbdharun's suggestions?

**Comment by kbdharun** [comment]
@EmilySeville7cfg If my suggestions are applicable can you apply them and merge this PR?

**Comment by EmilySeville7cfg** [comment]
> If the only issues holding up a merge are trivial fixes (typos, syntax errors, etc.), and the author doesn't respond in a day or two, maintainers can make the necessary changes themselves, and proceed with the merge process.

As rule is incomplete: `etc` is contained, I can consider this PR as trivial fix (adding missing Russian translation), apply your fixes and merge.

**Pull request merged**